server/container/guice/cassandra-guice/pom.xml

```
@@ -273,6 +278,124 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>install-jbpm</id>
```

```suggestion
                        <id>install-glowroot</id>
```
for all pom.xml in this PR :D